--- gettext-0.18.1.1/gettext-tools/gnulib-lib/spawn.in.h.orig	2011-07-13 19:56:35.000000000 -0400
+++ gettext-0.18.1.1/gettext-tools/gnulib-lib/spawn.in.h	2011-07-13 19:56:50.000000000 -0400
@@ -31,7 +31,7 @@
 
 /* Get definitions of 'struct sched_param' and 'sigset_t'.
    But avoid namespace pollution on glibc systems.  */
-#ifndef __GLIBC__
+#if 1 /*ndef __GLIBC__*/
 # include <sched.h>
 # include <signal.h>
 #endif
